.github/workflows/pr.yml

```diff
@@ -29,6 +29,8 @@ jobs:
 
       # Check for semvar violations
       - uses: obi1kenobi/cargo-semver-checks-action@v2
+        with:
+          exclude: moq-sub
```

Comment:
I probably need to land this on `main` in a separate PR first. Bah.

Reply:
Oh no, that worked. 
(Policy-wise that seems like a weird default to use the Action config from the branch you're on, but OK, whatever)